src/components/Accordion/Accordion.tsx

```diff
@@ -140,37 +166,38 @@ export const AccordionBody: FC<AccordionBodyProps> = ({
       role="region"
       {...rest}
     >
-      <div className={accordionBodyStyles}>{children}</div>
+      <div className={accordionBodyStyles}>{shouldRender && children}</div>
```

Comment:
do things break if you wrap this entire div?

Reply:
No, but no harm if we don't.